KillTeam/Controllers/TeamController.cs

```diff
@@ -0,0 +1,269 @@
+            _deleteTeamCommandHandler = deleteTeamCommandHandler;
+            _renameTeamCommandHandler = renameTeamCommandHandler;
+        }
+
+        private void InitializeToolbar(IList<ToolbarItem> toolbarItems)
+        {
+            ToolbarItems = toolbarItems;
+            ToolbarItems.Add(new ToolbarItem
+            {
+                Text = Properties.Resources.Tactiques,
+                Order = ToolbarItemOrder.Secondary,
+                Command = new Command(async () => await TacticsExecuted())
+            });
+            ToolbarItems.Add(new ToolbarItem
+            {
+                Text = Properties.Resources.Partager,
+                Order = ToolbarItemOrder.Secondary,
+                Command = new Command(async () => await ShareExecuted())
+            });
+
+            _errors = new ToolbarItem
+            {
+                Text = Properties.Resources.Erreurs,
+                Order = ToolbarItemOrder.Primary,
+                Icon = "Ressources/drawable/danger.png",
+                Command = new Command(async () => await ErrorsExecuted())
+            };
+        }
+
+        private void InitializeCommands()
+        {
+            AddMember = new Command(async () => await AddMemberExecuted());
+            ReorderMembers = new Command(ReorderMembersExecuted);
+            OpenMember = new Command(async e => await OpenMemberExecuted(e as TeamMemberViewModel));
+            DeleteMember = new Command(async e => await DeleteMemberExecuted(e as TeamMemberViewModel));
+            SelectMember = new Command(async e => await SelectMemberExecuted(e as TeamMemberViewModel));
+            EditName = new Command(async () => await EditNameExecuted());
+            EditRoster = new Command(async () => await EditRosterExecuted());
+            Delete = new Command(async () => await DeleteExecuted());
+
+
+        }
+
+        public async Task Refresh()
+        {
+            await UpdateItem();
+        }
+
+        private async Task UpdateItem()
+        {
+            var team = await KTContext.Db.Teams
+                .Where(e => e.Id == _itemId)
+                .Include(e => e.Members)
+                .ThenInclude(m => m.Specialist)
+                .Include(e => e.Members)
+                .ThenInclude(m => m.ModelProfile.Model.ModelWeapons)
+                .ThenInclude(fa => fa.Weapon)
+                .Include(e => e.Members)
+                .ThenInclude(m => m.ModelProfile.ModelProfileWeapons)
+                .ThenInclude(da => da.Weapon)
+                .Include(e => e.Members)
+                .ThenInclude(m => m.MemberWeapons)
+                .ThenInclude(m => m.Weapon)
+                .ThenInclude(a => a.WeaponProfiles)
+                .ThenInclude(a => a.WeaponType)
+                .Include(e => e.Members)
+                .ThenInclude(m => m.MemberWarGearOptions)
+                .ThenInclude(mr => mr.WarGearOption)
+                .Include(e => e.Faction)
+                .FirstAsync();
```

Comment:
I'd like to move these to "Queries" much like our Commands and CommandHandlers, so we can reuse these huge queries.